specifyweb/frontend/js_src/lib/components/Molecules/SvgIcon.tsx

```diff
@@ -61,6 +61,8 @@ export function SvgIcon({
       <g>
         <text
           dominantBaseline="central"
+          alignmentBaseline="middle"
+          baselineShift="-10%"
```

Comment:
The svg text does not vertically align without this for column headers. The `baselineShift` is a magic number but it perfectly aligns the text.  Without it, the vertically alignment is *juuust* so slightly to the top.

Without `alignmentBaseline` and `baselineShift`:
<img width="716" alt="Screenshot 2024-08-06 at 10 33 55 AM" src="https://github.com/user-attachments/assets/12d7c5a6-94c6-4328-952a-43e17e8f949e">

With both:
<img width="716" alt="Screenshot 2024-08-06 at 10 48 25 AM" src="https://github.com/user-attachments/assets/aa9bdae7-6404-451b-bbcf-6c106d915681">

Without `baselineShift`:
![image](https://github.com/user-attachments/assets/1c60115e-9a31-423b-9d28-b5b92f43cb51)

Reply:
I am ok with this fix, but I would like you to double check one thing:
The svg icons work fine everywhere else, but break in this one place.
I suspect thus the issue is not with the SVG icons, but with the interference with the handsontable styling.
could you verify if some style from handsontable is interfering with our icons? (use the styles panel in the dev tools for that)

then, if you find the cause, you can add the fixup CSS into this file:
https://github.com/specify/specify7/blob/production/specifyweb/frontend/js_src/css/workbench.css

I leave it up to you to pick which solution you prefer more